transaction-history-manager/src/main/java/com/qpay/transactionhistorymanager/service/impl/PdfReportGeneratorITextPdf.java

```diff
@@ -0,0 +1,217 @@
+    }
+
+    private String generatePath(final long userId) {
+        final DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd:hh:mm:ss");
+        final String currentDateTime = dateFormat.format(new Date());
+        return PDF_PATH + userId + "_" + currentDateTime + ".pdf";
+    }
+
+    private Table createTransactionsTable() {
+        final float [] pointColumnWidths = {50F, 150F, 100F, 150F, 250F};
+        final Table table = new Table(pointColumnWidths);
+        table.addCell("Number");
+        table.addCell("Date");
+        table.addCell("Transaction Type");
+        table.addCell("Amount of money");
+        table.addCell("Recipient/sender name");
+        return table;
+    }
+
+    private Paragraph createLogoParagraph() throws MalformedURLException {
+        final Paragraph logoParagraph = new Paragraph();
+        final ImageData imageData = ImageDataFactory.create(QPAY_LOGO_IMAGE_PATH);
+        final Image qpayLogoImage = new Image(imageData);
+        qpayLogoImage.scale(0.4F, 0.4F);
+        logoParagraph.add(qpayLogoImage);
+        return logoParagraph;
+    }
+
+    private Paragraph createTitleParagraph() {
+        final Paragraph titleParagraph = new Paragraph();
+        titleParagraph.setMarginTop(-75);
+        titleParagraph.setTextAlignment(TextAlignment.RIGHT);
+        final Text titleText = new Text("Activity summary");
+        titleText.setBold();
+        titleText.setFontSize(32);
+        titleParagraph.add(titleText);
+
+        return titleParagraph;
+    }
+
+    private Paragraph createSubTitleParagraph(final PdfReportInfo pdfReportInfo) {
+        final Paragraph subTitleParagraph = new Paragraph();
+        subTitleParagraph.setMarginTop(-10);
+        subTitleParagraph.setTextAlignment(TextAlignment.RIGHT);
+        final String periodStart = pdfReportInfo.periodStart().format(DateTimeFormatter.ofPattern("dd.MM.yyyy"));
+        final String periodEnd = pdfReportInfo.periodEnd().format(DateTimeFormatter.ofPattern("dd.MM.yyyy"));
+        final Text subTitleText = new Text("For period from " + periodStart + " to " + periodEnd);
+        subTitleText.setFontSize(14);
+
+        subTitleParagraph.add(subTitleText);
+
+        return subTitleParagraph;
+    }
+
+    private Paragraph createClientNameParagraph(final String name) {
+        final Paragraph clientNameParagraph = new Paragraph();
+        clientNameParagraph.setMarginTop(15);
+        final Text clientText = new Text("Client: ");
+        final Text nameText = new Text(name);
+        nameText.setBold();
+        clientNameParagraph.addAll(List.of(
+                clientText,
+                nameText
+        ));
+        return clientNameParagraph;
+    }
+
+    private Paragraph createTotalPaymentsReceivedParagraph(final BigDecimal total) {
+        final Paragraph totalPaymentsReceivedParagraph = new Paragraph();
+        final Text totalPaymentsText = new Text("Total payments received: ");
```

Comment:
```suggestion
        final Text totalPaymentsText = new Text("Total income: ");
```
again, let's make it more generic